CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ _None._
 
 -->
 
-## Unreleased
+## 12.0.0
```

Comment:
A note to me and reviewers: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/706 bumped the version but didn't updated the changelog file.